src/main/webapp/main/LandingPage.js

```diff
@@ -119,15 +124,19 @@ const LandingPage = hh(class LandingPage extends Component{
         projectList: projectList,
         taskList: taskList,
         logged: user.data.session
-      })
+      });
+      Storage.setUserIsLogged(user.data.session);
     }
   };
 
   render() {
+    if (Storage.userIsLogged() && !this.state.logged) {
+      this.init();
+    }
```

Comment:
This is already in `componentDidMount` ... do we need to call it a second time before rendering?

Reply:
Greg, When user is not logged, the component is been created with "logged = false". After user logs in, we display the same page and "logged" is still false.  We need some way to change the state of this component.

Reply:
The main problem is that we're doing this in the render. Render is called every time state changes. The init method changes state, so you get into an infinite loop situation. You need to handle the state changing events in the appropriate lifecycle handlers.